### DISCUSSION TITLE
Learning programming

### DISCUSSION THREAD

**Discussion by Aws-pixel** [3b1b/manim#2040] [Q&A]
I am completely new here. I just want to ask if there's a place to learn programming here?

**Comment by ruslansenatorov** [comment]
> I am completely new here. I just want to ask if there's a place to learn programming here?

yes @SENATOROVAI

**Comment by Ayushxxz** [comment]
> > I am completely new here. I just want to ask if there's a place to learn programming here?
> 
> 
> 
> yes @SENATOROVAI

Could you tell me I wanna learn to code
